flocx_market/db/sqlalchemy/api.py

```diff
@@ -1,36 +1,103 @@
+from oslo_db.sqlalchemy import session as db_session
 from oslo_utils import uuidutils
 
-from flocx_market.db.orm import orm
+import flocx_market.conf
 from flocx_market.db.sqlalchemy import models
 
 
+CONF = flocx_market.conf.CONF
+_engine_facade = None
+
+
+def get_facade():
+    global _engine_facade
+    if not _engine_facade:
+        _engine_facade = db_session.EngineFacade.from_config(CONF)
```

Comment:
What's an `EngineFacade` for?

Reply:
It's for database session handling: https://docs.openstack.org/oslo.db/latest/user/usage.html#session-handling